_config.yml

```diff
@@ -254,17 +254,17 @@ kramdown:
     coderay_css: style
 
 text:
-  ## Values last updated 2017-05-21
+  ## Values last updated 2018-10-16
   ## All variable names must indicate unit type for easy translation of adjacent text,
   ##   such as: subsidy_in_decimal_bitcoins or
   ##   bitcoin_org_docs_maintainer_email_link
   ## For increasing variables, like chain size, choose a somewhat higher
   ##   value so we don't need to update too often
   subsidy_in_decimal_bitcoins: 12.5
   ## cd ~ ; du -sh .bitcoin/ --exclude=testnet3 --exclude=regtest
-  bitcoin_datadir_gb: 145
+  bitcoin_datadir_gb: 200
```

Comment:
For future reference: Add a comment with the shell command used to produce this number?